src/post/postSingle/PostSingleComments.less

```diff
@@ -1,7 +1,12 @@
+
 .PostSingleComments {
   font-size: 1.2em;
   border-top: 1px solid #E7E7EC;
 
+  container {
```

Comment:
This class is not using BEM and also missing the dot in prefix ".". It's not seem to be used anyway, you can remove it.